src/cli/preprocessors/mod.rs

```diff
@@ -17,38 +17,38 @@ impl Preprocessors for Command {
             .next_help_heading("Preprocessors")
             .args([
                 #[cfg(feature = "resize")]
-                arg!(--resize <RESIZE> "Resize the image(s) according to the specified criteria")
-                    .long_help(indoc! {"Resize the image(s) according to the specified criteria
+                arg!(--resize <RESIZE> "Resize the image(s) according to the specified criteria.")
+                    .long_help(indoc! {r#"Resize the image(s) according to the specified criteria.
 
                     Possible values:
                     - @1.5:    Enlarge image size by this multiplier
                     - 150%:    Adjust image size by this percentage
                     - 100x100: Resize image to these dimensions
-                    - 200x_:   Adjust image dimensions while maintaining the aspect ratio based on the specified dimension"})
+                    - 200x_:   Adjust image dimensions while maintaining the aspect ratio based on the specified dimension"#})
                     .value_parser(value_parser!(ResizeValue))
                     .action(ArgAction::Append),
 
                 #[cfg(feature = "resize")]
-                arg!(--filter <FILTER> "Filter that used when resizing an image")
+                arg!(--filter <FILTER> "Filter that used when resizing an image.")
                     .value_parser(value_parser!(ResizeFilter))
                     .default_value("lanczos3")
                     .requires("resize"),
 
                 #[cfg(feature = "quantization")]
-                arg!(--quantization [QUALITY] "Enables quantization with optional quality")
-                    .long_help(indoc! {"Enables quantization with optional quality
+                arg!(--quantization [QUALITY] "Enables quantization with optional quality.")
```

Comment:
maybe add that this value is in percentages?

Reply:
Changed, but the sentense is still really strange...